cabby/model/text/util.py

```diff
@@ -70,6 +85,27 @@ def load_checkpoint(load_path: Text, model:  torch.nn.Module,
   return state_dict
 
 
+
+def save_metrics_last_only(save_path: Text,
+         true_points_list: Sequence[Tuple[float, float]],
+         pred_points_list: Sequence[Tuple[float, float]]):
+  '''Function for saving results.'''
+
+  if save_path == None:
+    return
+  state_dict = {
+          # 'index': list(range(len(true_points_list))),
```

Comment:
keep the index or some other example identifier?

Reply:
I will remove the comment. I output the index of the DataFrame